app/src/pages/inside/stepPage/modals/makeDecisionModal/executionSection/optionsSection/itemsList/itemsListHeader/itemsListHeader.jsx

```diff
@@ -37,26 +37,23 @@ export const ItemsListHeader = ({
   onShowErrorLogsChange,
   optionValue,
   isNarrowView,
-  isBulkOperation,
   eventsInfo,
 }) => {
   const { formatMessage } = useIntl();
   const { trackEvent } = useTracking();
   const [isAllSelected, setIsAllSelected] = useState(selectedItems.length === testItems.length);
-  const defectFromTIGroup =
-    testItems.length > 0 && testItems[0].issue.issueType.startsWith(TO_INVESTIGATE_LOCATOR_PREFIX);
+  const defectFromTIGroup = (
+    (!isEmptyObject(currentTestItem) && currentTestItem) ||
```

Comment:
May be swap these two conditions?
`(currentTestItem && !isEmptyObject(currentTestItem)) || ...`

Reply:
It doesn't work because `&&` operator return last operand in case of all operands before was true

Reply:
You're right, but such implementation looks a bit tricky